src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.test.tsx

```diff
@@ -78,7 +78,7 @@ describe('<BlastInputSequence />', () => {
       const { container } = render(<BlastInputSequence {...commonProps} />);
       const deleteButton = container.querySelector('.deleteButton');
 
-      expect(deleteButton).toBe(null);
+      expect(deleteButton).toBeTruthy();
```

Comment:
`does not contain a` -> `does contain a` ?

Reply:
✅